package.json

```diff
@@ -2,22 +2,34 @@
   "name": "funcadelic",
   "version": "0.3.0",
   "description": "Functional Fundamentals are Fun!",
-  "main": "nodule.js",
-  "module": "src/funcadelic.js",
+  "main": "dist/funcadelic.cjs.js",
+  "module": "dist/funcadelic.esm.js",
```

Comment:
Any reason not to point `module` at `src/funcadelic.js`? It feels like it ought to be possible to include funcadelic in a build system that has never been touched by `rollup`.  In other words, it feels natural that there should be _some_ entry point specified for the pure source.

Reply:
rollups creates one file for all modules which is faster to parse and easier to navigate as a dependency. It seems like a reasonable default. I'll leave it as is if you don't object.